### DISCUSSION TITLE
Auto Checkout bot recommendation?

### DISCUSSION THREAD

**Discussion by Contact2020** [jef/streetmerchant#1632] [General]
Hey guys,

I'm looking for a bot that can auto checkout so I can purchase some rtx 3080s for mining.
Playing with this for a bit and it looks like it can add to cart , but it's on me to be ready at all times to complete a transaction.

Are there any auto buying bots that you'd recommend?

**Comment by footloops** [comment]
The fact that you're planning on using the 3080's to mine, just makes me and probably others not to recommend you any sort of bot that can do that. Personally to me, it's just scummy to get a product that is in high demand just to mine bitcoins, when there are others out there just trying to get one to finish a build. If you really want that 3080 to mine, then you better be ready to sunk hours into getting it.

To answer your question though, the auto add to cart function I believe is only for the Nvidia site.

*Again, my opinion. If it appears as if I'm attacking you, then I'm sorry. It isn't the intent of this comment.

**Reply by Contact2020** [reply]
Hey that's a good point, and I wouldn't want to seem scummy. I just kind of look at gpus as tools that can either make games look better or can be used to pay your mortgage. Both ideologies are certainly valid, just a question of priorities.    

Though I feel you are correct and would delete the question if I could.

**Reply by footloops** [reply]
Fair point. Glad to have this little chat with no animosity.

**Comment by lec6213** [comment]
Might want to get off ye high horse there, its the makers that are back door selling every card they produce to the miners.  Don't blame the miners who are buying the cards just becasue the maker wants to sell 500 at a pop versus one at a time here and there to gamers.  In the end its all about money, as almost everything in this world is and the manufacturers are choosing the money route over those of us that want only 1 or can only afford 1 card.  

If I am getting beat out of buying a card then someone else it doing it better/faster then I am and that is one me.  This lets blame the consumer, just becasue of what they choose to do with their purchase BS it really old....find a new line.

And this is why folks don't ask questions or offer help, if you cant help or simply wont becasue of your own misinformation then don't.

Contact2020, if I knew I would help you out but the one I did try did not work out for me, good luck.

**Reply by footloops** [reply]
1) I'm not blaming anyone.
2) Not blaming any consumer
3) Never told him to stop doing it.
4) All I said was that I personally feel its a scummy and unfair move.

All I did was state my opinion on the matter of the current mining craze going around and how to me, it seems to be unfair to everyone else just trying to finish a build if they were to use an auto checkout.
